templates/aruba_os_show_arp.textfsm

```diff
@@ -0,0 +1,6 @@
+Value IP_ADDRESS (\d+.\d+.\d+.\d+)
+Value HARDWARE_ADDRESS (\w+.\w+.\w+.\w+.\w+.\w+)
+Value INTERFACE (\w+)
```

Comment:
Do you mind aligning the values with other ARP related templates? Such as `cisco_ios_show_ip_arp.textfsm`

```suggestion
Value ADDRESS (\d+.\d+.\d+.\d+)
Value MAC (\S+)
Value INTERFACE (\S+)
```